src/Components/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Checkbox, Form, Input } from "antd";
+import React, { useContext } from "react";
+import { StandardButton, TravelDiv } from "./Style";
+import { useGlobalContext } from './UserContext'
+import { useNavigate } from "react-router-dom";
+
+
+
+const Login: React.FC = () => {
+  const { user, setUser } = useGlobalContext()
+  const navigate = useNavigate();
+
+
+  const onFinish = (e: { username: string; }) => {
+    console.log(e.username);
+    setUser(e.username);
+    navigate("/");
+  };
+
+  const onFinishFailed = (errorInfo: any) => {
+    console.log("Failed:", errorInfo);
```

Comment:
Csinalhatsz esetleg egy kondicionalis renderelest ebben az esetben, hogy a user fele is kommunikald a problemat.